modules/apps/list-type/list-type-service/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay List Type Service
 Bundle-SymbolicName: com.liferay.list.type.service
 Bundle-Version: 1.0.41
-Liferay-Require-SchemaVersion: 1.3.0
+Liferay-Require-SchemaVersion: 1.4.0
```

Comment:
Is it correct that updating only data we need to increase the minor version and not only the patch one? like 1.3.1